SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -45,33 +36,24 @@ public static Crashes Instance
             {
                 lock (CrashesLock)
                 {
-                    _instanceField = value;
+                    _instanceField = value; //for testing
```

Comment:
For testing?

Reply:
Its the whole setter that's for unit tests.
It's a copy/paste from Analytics.
We should move the comment to set or consider making it internal, and use InternalsVisibleTo the test assembly.